README.md

```diff
@@ -5,6 +5,20 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/openshift/assisted-service)](https://goreportcard.com/report/github.com/openshift/assisted-service)
 [![License Apache](https://img.shields.io/github/license/openshift/assisted-service)](https://opensource.org/licenses/Apache-2.0)
 
+## About
+
+This repository provides a service that installs OpenShift.  The service exposes either a REST API, or it can be deployed as an Operator where it exposes a Kubernetes-native API via Custom Resources.  A [UI](https://github.com/openshift-assisted/assisted-ui) is available that uses the REST API.
+
+The Assisted Service can currently install clusters with highly-available control planes (3 hosts and above) and can also install Single-Node OpenShift (SNO).  Highly available clusters are configured to use OpenShift's `bare-metal` platform, while SNO uses `none`.
+
+The basic flow for creating a new OpenShift cluster using the Assisted Service via the UI or REST API is:
+1. Create a new Cluster resource with the minimal required properties.
+1. Generate and download a bootable image which is customized for that cluster. This image is based on RHCOS and is customized to automatically run an agent upon boot.
+1. Boot the hosts that will form the cluster with the image from the previous step. The boot method is left to the user (e.g., USB drive, virtual media, PXE, etc.).
+1. The agent running on each host contacts the Assisted Service via REST API and performs discovery (sends hardware inventory and connectivity information).
+1. The UI guides the user through the installation, with the service performing validations along the way.  Alternatively, this can be done via API.
+1. Once all validations pass, the user may initiate the installation.  Progress may be viewed via the UI or API, and logs are made available for download.
```

Comment:
I would add also the following with relevant links:
- How does the installation happen? (wipe, writing image to disk using coreos-installer, reboot)
- How do we finish an installation? (Looking on CVO status)

Reply:
I don't want to get into this level of detail here.  The point is to give someone a high-level idea of what this software does.  These items would be useful for another doc file.